### ISSUE TITLE
The density mark could support the density keyword for fillOpacity and strokeOpacity, too

### ISSUE THREAD

**Issue by mbostock** [observablehq/plot#1399]
It’s currently only supported for **stroke** and **fill**:

https://github.com/observablehq/plot/blob/02731fb97277442120cb0d64350e60e6665c2ad1/src/marks/density.js#L24-L29

**Comment by Fil** [comment]
For fillOpacity the tricky thing is that the polygons are layered (as opposed to what we would have with isobands), so the opacities will be compounded. https://github.com/d3/d3-contour/issues/71

**Comment by mbostock** [comment]
Agreed. I think we should switch to generating isobands by default.

**Comment by mbostock** [comment]
And strokeWidth.
